lib/rig_inbound_gateway_web/v1/connection_controller.ex

```diff
@@ -0,0 +1,74 @@
+defmodule RigInboundGatewayWeb.V1.ConnectionController do
+  @moduledoc false
+  use Rig.Config, [:cors]
+  use RigInboundGatewayWeb, :controller
+  use RigInboundGatewayWeb.Cors, :cors
+  use RigInboundGatewayWeb.Cors, :preflight_put
+
+  alias Result
+  alias Rig.Connection.Codec
+  alias RigInboundGatewayWeb.VConnection
+
+  require Logger
+
+  @heartbeat_interval_ms 15_000
+  @subscription_refresh_interval_ms 60_000
+
+  @doc """
+  ### Dirty Testing
+
+      CONN_TOKEN=$(http :4000/_rig/v1/connection/init)
+      http --stream ":4000/_rig/v1/connection/sse?connection_token=$CONN_TOKEN"
+  """
+  @spec init(conn :: Plug.Conn.t(), params :: map) :: Plug.Conn.t()
+  def init(%{method: "GET"} = conn, _) do
+    {:ok, vconnection_pid} = VConnection.start(@heartbeat_interval_ms, @subscription_refresh_interval_ms)
+    send vconnection_pid, :timeout
```

Comment:
`:timeout` is also the message that signals a GenServer timeout => please use a different atom for this to avoid confusion

Reply:
How about `:vconnection_timeout`?

Reply:
fine for me